.github/workflows/cross-compile.yml

```diff
@@ -0,0 +1,19 @@
+on: [push, pull_request]
+jobs:
+  crosscompile:
+    strategy:
+      matrix:
+        go: [ "1.14", "1.15" ]
```

Comment:
beware that this is 1.14.0, not the latest 1.14.x. You should use `[ "1.14.x", "1.15.x" ]`.